xpdacq/glbl.py

```diff
@@ -79,39 +61,92 @@ class glbl():
     yaml_dir = YAML_DIR
     allfolders = ALL_FOLDERS
     archive_dir = USER_BACKUP_DIR
-    beamhost = BEAMLINE_HOST_NAME
     dk_yaml = DARK_YAML_NAME
     dk_window = DARK_WINDOW
     frame_acq_time = FRAME_ACQUIRE_TIME
     auto_dark = True
+    owner = OWNER
+    beamline_id = BEAMLINE_ID
+    group = GROUP
+    ''' this block of code should be taken care in the following block of code. Delete it after testing
+    # objects for collection activities
+    Msg = None
+    xpdRE = None
+    Count = None
+    AbsScanPlan = None
+
     area_det = None
     shutter = None
     LiveTable = None
+    temp_controller = None
 
+    # objects for analysis activities
     db = None
     get_events = None
     get_images = None
     verify_files_saved = None
-
-    xpdRE = RunEngine()
-    xpdRE.md['owner'] = OWNER
-    xpdRE.md['beamline_id'] = BEAMLINE_ID
-    xpdRE.md['group'] = GROUP
-    
-    if hostname != BEAMLINE_HOST_NAME:
-        shutter = mock_shutter()
+    '''
+    # logic to assign correct objects depends on simulation or real experiment
+    hostname = socket.gethostname()
+    if hostname == BEAMLINE_HOST_NAME:
```

Comment:
This is a _super_ sketchy/brittle way to do this.  A better way might to look for an env like `'XPD_REAL_COLLECTION_ENV'` environmental variable that you can make sure you set as part of a bash script you use to launch ipython